### ISSUE TITLE
Is memory optimization supported?

### ISSUE THREAD

**Issue by HolmesShuan** [OAID/Tengine#35]
The speed on RK3399 is **impressive** and significantly outperform ncnn. Did Tengine further equip with memory reuse or computation graph optimizations? If so, did Tengine-based Caffe/TF benefits from this technique? Memory overuse has been a long-term issue in Caffe.

**Comment by lyuchuny3** [comment]
Yes, Tengine supports memory reuse and graph optimization. Tengine-based Caffe/TF can benefit from this technique.
